frontend/src/components/ui/alert-dialog.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import * as AlertDialogPrimitive from '@radix-ui/react-alert-dialog';
+import { cn } from '../lib/utils';
+import { buttonVariants } from './button';
+
+const AlertDialog = AlertDialogPrimitive.Root;
+
+const AlertDialogTrigger = AlertDialogPrimitive.Trigger;
+
+const AlertDialogPortal = AlertDialogPrimitive.Portal;
+
+const AlertDialogOverlay = React.forwardRef<
```

Comment:
Nitpick:
I'd recommend to use Dialog instead
I personally do not like alert dialog cause it doesn't close when the customer clicks out of it and it doesn't have a close cross button. 
The only way to close it is to add 

So you needed to make a hack of adding the close button yourself, while dialog has it there by default and it supports clicking out of the modal to close it

Reply:
Done!